packages/core/src/Badge/Badge.stories.tsx

```diff
@@ -157,6 +158,22 @@ export const Test: StoryObj = {
       <HvBadge showCount label={8}>
         <HvTypography variant="title4">Events</HvTypography>
       </HvBadge>
+      <HvBadge variant="subtle" count={10} icon={<Alert />} />
+      <HvBadge variant="subtle" showCount count={8} icon={<Alert />} />
+      <HvBadge variant="subtle" showCount label={88} icon={<Alert />} />
+      <HvBadge variant="subtle" showCount label={888} icon={<Alert />} />
+      <HvBadge variant="subtle" count={0} text="Events" textVariant="label" />
+      <HvBadge variant="subtle" label={10} text="Events" />
+      <HvBadge
+        variant="subtle"
+        showCount
+        label={10}
+        maxCount={5}
+        text="Events"
+      />
```

Comment:
There's no need to duplicate the whole API - especially the deprecated props
```suggestion
      <HvBadge variant="subtle" showCount label={88} icon={<Alert />} />
```